website/vcd.erb

```diff
@@ -13,6 +13,9 @@
         <li<%= sidebar_current("docs-vcd-resource") %>>
           <a href="#">Resources</a>
           <ul class="nav nav-visible">
+            <li<%= sidebar_current("docs-vcd-resource-catalog-item") %>>
+              <a href="/docs/providers/vcd/r/catalog_item.html">vcd_catalog_item</a>
```

Comment:
After merge, please make sure that vcd_catalog_item and vcd_catalog_media entries are listed one after another.